contracts/src/Params.sol

```diff
@@ -82,3 +82,15 @@ struct SetPricingParametersParams {
     /// @dev Fee multiplier
     UD60x18 multiplier;
 }
+
+// Payload for TransactCall
+struct TransactCallParams {
+    /// @dev The agent ID of the consensus system
+    bytes32 agentID;
+    /// @dev The target contract
+    address target;
+    /// @dev Payload of the call
+    bytes payload;
+    /// @dev Max gas cost of the call
+    uint64 dynamicGas;
```

Comment:
The message packet already has a `maxDispatchGas`, which is the upper limit a transact can consume. So I am not sure what role this `dynamicGas` plays.

The gas estimator on BridgeHub should add a buffer to `maxDispatchGas` to cover the cost of dispatching via the agent.

Reply:
Yes, it's the same thing. Just rename it for less confusion.
https://github.com/Snowfork/snowbridge/pull/1145/commits/27c14e94cdba7eed261178b7f2391b055d74bee5